test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -72,17 +72,45 @@ void InterpreterState::dumpTraceAndState(ostream& _out, bool _disableMemoryTrace
 	}
 	_out << "Storage dump:" << endl;
 	dumpStorage(_out);
+
+	if (!calldata.empty())
+	{
+		_out << "Calldata dump:";
+
+		for (size_t offset = 0; offset < calldata.size(); ++offset)
```

Comment:
Could you please dump only non-zero values like this?

https://github.com/ethereum/solidity/blob/f7fcd8ccaac2dadb14761198e980305de00d2baa/test/tools/yulInterpreter/Interpreter.cpp#L66-L71